tests/rados/test_osd_replicated_inconsistency_scenario.py

```diff
@@ -97,12 +98,26 @@ def run(ceph_cluster, **kw):
         auto_repair_value = mon_obj.get_config(
             section="osd", param="osd_scrub_auto_repair"
         )
-        log.info(f"Original value of osd_scrub_auto_repair is {auto_repair_value}")
+        msg_auto_repair = (
+            f"Original value of osd_scrub_auto_repair is {auto_repair_value}"
+        )
+        log.info(msg_auto_repair)
         if auto_repair_value is True:
             log.error("The default value of the osd_scrub_auto_repair should be false")
             return 1
 
+        log.info("====VERIFICATION OF THE TESTS BY PERFORMING SCRUB OPERATIONS====")
+
         # case 1: inconsistent objects are greater than the osd_scrub_auto_repair_num_errors count
+        log.info(
+            "Test scenario1:Inconsistent objects greater than the osd_scrub_auto_repair_num_errors count\n"
+            + "operation : scrub\n"
+            + "Expectation : No repairs to be made"
+        )
+        scrub_error_count = get_scrub_error_count(rados_obj)
```

Comment:
We have set no deep scrub on the cluster. The method we're calling `get_scrub_error_count`, by default it runs the deep-scrub. how would it run and complete? This might not work as expected here.

I feel that we should write another method for getting the scrub error count, one that does not involve scrubs/ deep scrubs.

We should only run scrub/deep scrub only to identify or repair the Inconsistency.